sql/src/test/java/io/deephaven/sql/SqlAdapterTest.java

```diff
@@ -0,0 +1,309 @@
+package io.deephaven.sql;
+
+import io.deephaven.qst.column.header.ColumnHeader;
+import io.deephaven.qst.table.Graphviz;
+import io.deephaven.qst.table.TableHeader;
+import io.deephaven.qst.table.TableSpec;
+import io.deephaven.qst.table.TicketTable;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Moving from numbers to descriptive names would make this better and more maintainable.

Reply:
or maybe named constants in the test class